core/src/runtime.ts

```diff
@@ -89,6 +79,11 @@ export const createCapacitor = (win: WindowCapacitor): CapacitorInstance => {
           typeof jsImplementations[platform] === 'function'
             ? (jsImplementation = await jsImplementations[platform]())
             : (jsImplementation = jsImplementations[platform]);
+      } else if (capCustomPlatform?.useWebFallbackPlugins && !jsImplementation && 'web' in jsImplementations) {
```

Comment:
this seem to do the same as the if. isn't electron platform in jsImplementations?

Reply:
Without this you will get: `Error: "Network" plugin is not implemented on electron`. Which is fine too but this forces web versions to load however it should read `else if (capCustomPlatform !== null && !jsImplementation && 'web' in jsImplementations)`. This is only relevant to custom platforms (I don't know what android and iOS behavior would be with web plugins being forced loaded.)

Reply:
Will submit new commit to show all these adjustments if you could review it again after that commit?